### DISCUSSION TITLE
Add option to block keywords of only a certain website among a blocklist

### DISCUSSION THREAD

**Discussion by Tim676o** [proginosko/LeechBlockNG#412] [Ideas]
Currently, when I place a website under another one with a few keywords, it is automatically brought up to the top of the blocklist, and so it doesn't block it until there's one of the keywords and I have to create a new Block.

**Comment by proginosko** [comment]
LB automatically sorts the list of sites alphabetically, so that it's easy to find an entry in a long list. The order of sites makes no difference to the blocking behavior.

"so it doesn't block it until there's one of the keywords" -- yes, that's the intended behavior. See the FAQ page on keyword blocking.

**Reply by Tim676o** [reply]
Right. Is there an option to block the keywords of only one website of a list where there are other websites before that one though for a same block ? If not then I make this discussion so maybe you could one day, if you like to, make that option accessible

**Reply by proginosko** [reply]
You would need to use two block sets for that: one with the keyword(s), one without.
